cli.js

```diff
@@ -116,6 +116,11 @@ if (cli.flags.init) {
 	return;
 }
 
+if (cli.flags.watch && cli.flags.tap && !conf.tap) {
```

Comment:
Where would `hasFlag()` come from? I didn't look too closely at `meow` but I assumed I couldn't really see whether `cli.flags.tap` came from `--tap` or `conf`. Probably not a good idea to reparse `argv` either.



> ... Or I'm just overthinking it.

`--tap` would be superfluous if it's already in the `package.json`, and there won't be an error in the `package.json` anyway, so I don't think this is a big deal.